packages/regenesis-surgery/scripts/data.ts

```diff
@@ -140,20 +169,17 @@ export const loadSurgeryData = async (
   const ropstenProvider = new ethers.providers.JsonRpcProvider(
     configs.ropstenProviderUrl
   )
+
   const ropstenWallet = new ethers.Wallet(
     configs.ropstenPrivateKey,
     ropstenProvider
   )
-
-  // Get a reference to the L1 provider.
-  console.log('Connecting to L1 provider...')
-  const l1Provider = new ethers.providers.JsonRpcProvider(configs.l1ProviderUrl)
-
-  // Get a reference to an ETH (mainnet) provider.
-  console.log('Connecting to ETH provider...')
-  const ethProvider = new ethers.providers.JsonRpcProvider(
-    configs.ethProviderUrl
-  )
+  const ropstenChainId = await ropstenWallet.getChainId()
+  if (ropstenChainId !== 3) {
+    throw new Error(
+      `Ropsten chain id incorrect, got ${ropstenChainId} and expected 3`
```

Comment:
i've been secretly using kovan for this provider since i don't have ropsten eth 😅 but also ok with switching it

Reply:
I feel like we should use kovan here because we have more kovan eth in general and its also not proof of work which may have massive reorgs at the wrong time

Reply:
We were using ropsten because it's a Geth network and Uniswap asked for us to use a Geth-based network for this